packages/node/src/evm/templates/template-fetching.ts

```diff
@@ -53,6 +52,7 @@ async function fetchTemplateGroup(
   if (err || !rawTemplates) {
     const groupLog = logger.pend('ERROR', 'Failed to fetch API call templates', err);
 
+    // TODO: is this still needed now that getTemplate() was removed from protocol?
```

Comment:
I'm not quite sure why `getTemplate()` was removed from the protocol, but if it doesn't exist anymore and will throw an error if called, then we should definitely remove it

Reply:
It's now called [`templates()`](https://github.com/api3dao/airnode/blob/3dfd88a627e058fb42883258ad755c1828d500d8/packages/protocol/contracts/rrp/interfaces/ITemplateUtils.sol#L27) because we're using the default getter

Reply:
Right, we can use the default getter now or we can use the [getTemplates()](https://github.com/api3dao/airnode/blob/fb1be6ab4e1952e359ed4858de5340346f07980c/packages/protocol/contracts/rrp/TemplateUtils.sol#L53) function if we need to pass a list of templateIds as argument.

The thing here is that I think the fallback mechanism now makes no sense since we would be calling the same function. Previously if getTemplates() call failed then we would call getTemplate(). Now we only have getTemplates(). So I think I'll just remove the fallback code and keep the error.

Reply:
Yeah I think that's fine then. The `getTemplates()` function is already retried once if it fails anyway

Reply:
I've created a separate [issue](https://api3dao.atlassian.net/browse/AN-118) for doing this refactor so I can merge this PR.

Reply:
I don't get the problem here. Previously we fell back from `getTemplates()` to `getTemplate()`. Now we'll fall back from `getTemplates()` to `templates()`. There was only supposed to be a naming change.

Reply:
Ok I misunderstood, sorry. Let's just switch to `templates()` instead then